packages/theme/src/index.ts

```diff
@@ -11,9 +11,9 @@ governing permissions and limitations under the License.
 */
 export * from './theme.js';
 
-import './theme-dark.js';
-import './theme-darkest.js';
 import './theme-light.js';
```

Comment:
'light' in the default in the tests, so this ensure that it is still `default` by loading it first.